### DISCUSSION TITLE
When using lower frame rates like 1 or 2 FPS with WebRTC experiencing connection issues.

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#3915] [Questions]
### Question

When using lower frame rates like 1 or 2 FPS with WebRTC experiencing connection issues.

**Comment by aler9** [comment]
Hello, from my experience, most encoders are not even capable of producing such low frame rates. FFmpeg and libx264 surely are not capable of that. I can't say anything else since you didn't provide a sample source, server logs, browser logs or anything that allows to reproduce the specific issue.
